docs/pipeline/bssh_sqs_event_replay.json

```diff
@@ -0,0 +1,64 @@
+{
+  "Records": [
+    {
+      "messageId": "c3c8ac12-4e87-4d8d-9b04-caeb3e8ee4a1",
+      "receiptHandle": "AQEBsqJzRkofws0JNZJUnK7O5+9MGlTFem6yoTqYxJOq3JfULa5HX9M3yEjNJBaTW7Hd1L/gZ/txSZ6yUslsntSJfzvnW2ZLISc1FObz6su54g79AlaIgLZjlEPQ7Zjyq3wko8RRNWpFxDIkuffe0WrrmBlrYFUB/5pnBUFl4L9YeZwfLFgUiP6tpFCBsfomlagXrkdhqzddbYr4n0m4kmblURsmk3/ys+8lJNEWjMs3olggdazCSGMZ9OO/B+lbifps2UiqDbXDcCwJ84xeIYgluWAsggb6KVKElIp2v0sQRVphxUC7mRW4+/8JFlhiBv5nvdCg8WlefvFdq2IJLHPouphJBOEuGxe1trc838o44zpz3PkB8Nmuut3FavH/+CR7wzQC7hB47wnFY78MHUF0YGzVi9huw462npN+4gfJQgI=",
+      "body": "{\"gdsFolderPath\":\"/200612_A01052_0017_BH5LYWDSXY\",\"gdsVolumeName\":\"umccr-raw-sequence-data-dev\",\"reagentBarcode\":\"null\",\"v1pre3Id\":\"623627\",\"dateModified\":\"2021-04-12T04:29:51.4800606Z\",\"acl\":[\"wid:9c481003-f453-3ff2-bffa-ae153b1ee565\"],\"flowcellBarcode\":\"1ADD1CKMC\",\"sampleSheetName\":\"SampleSheet.csv\",\"apiUrl\":\"https://api.aps2.sh.basespace.illumina.com/v2/runs/r.AnPk0ox1OU-pACGtvSEQC2\",\"name\":\"200612_A01052_0017_BH5LYWDSXY\",\"id\":\"r.AnPk0ox1OU-pACGtvSEQC2\",\"instrumentRunId\":\"200612_A01052_0017_BH5LYWDSXY\",\"status\":\"PendingAnalysis\"}",
+      "attributes": {
+        "ApproximateReceiveCount": "1",
+        "SentTimestamp": "1589509372323",
+        "SenderId": "AROARFCPI2IGZA4XHGYYY:kiam-kiam",
+        "ApproximateFirstReceiveTimestamp": "1589509373008"
+      },
+      "messageAttributes": {
+        "subscription-urn": {
+          "stringValue": "urn:ilmn:igp:us-east-1:YXdzLXVzLXBsYXRmb3JtOjEwMDAwNTM3OjBiYTU5YWUxLWZkYWUtNDNiYS1hM2I1LTRkMzY3YTQzYWJkNQ:subscription:sub.889",
```

Comment:
This is our tenant ID, right?
I don't think exposing this is a security risk though.

Reply:
Right. Oversight, I better rotate it.